src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -616,23 +615,27 @@ void style_transfer::train(gl_sarray style, gl_sarray content,
 
 std::shared_ptr<MLModelWrapper> style_transfer::export_to_coreml(
     std::string filename, std::map<std::string, flexible_type> opts) {
-  flex_int image_width = read_state<flex_int>("image_width");
-  flex_int image_height = read_state<flex_int>("image_height");
+  const flex_int image_width =
+      read_opts<flex_int>(opts, "image_width", /* delete item */ true);
```

Comment:
Why are we deleting these keys from the dictionary?

Reply:
I guess because we don't need them anymore so I delete them.